deploy/docker/pump.io.json

```diff
@@ -0,0 +1,23 @@
+{
+    "driver":  "mongodb",
+    "params":  {"dbname": "pumpio", "host":"127.0.0.1", "port":27017},
+    "secret":  "put a secret string here",
```

Comment:
Aren't all these config values supposed to be configurable by whoever's installing the Docker image?